1001-2000/LIT1558Matthew.xml

```diff
@@ -68,26 +71,63 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                 እምነገደ፡ ይሳኮር፡ እምሀገረ፡ ናዝሬት። ስመ፡ አቡሁ፡ ዲቁ፡ ወስመ፡ እሙ፡ ካሩትያስ።</ab></div>
+                <div type="textpart" subtype="explicit"><ab>ወሰበከ፡ ባቲ፡ በህንድ፡ ወበኢየሩሳሌም።</ab></div>
+             </div>
+             <div type="textpart" subtype="textsubunit" xml:lang="gez" xml:id="EusebianSectionsMatthew">
+                <label>Statement about Eusebian Sections in the Gospel of Matthew</label>
+                <div type="textpart" subtype="incipit"><ab>ወኍልቈ፡ ምዕራፋቲሁ፡ ንኡሳን፡ ፫፻፶፭፡</ab></div>
+                <div type="textpart" subtype="explicit"><ab>ወምዕራፉ፡ በልሳነ፡ ሮሜ። ፷፰፡</ab></div>
+             </div>
+             <div type="textpart" subtype="textsubunit" xml:lang="gez" xml:id="TituliMatthew">
+                <label>Tituli of the Gospel of Matthew</label>
+                <div type="textpart" subtype="incipit"><ab>
+                <l n="1">፩፡ ሰብአ፡ ሰገል።</l>
+                <l n="2">፪፡ ቅትለተ፡ ሕፃናት።</l>
+                <l n="3">፫፡ ስብከተ፡ ዮሓንስ።</l></ab></div>
+                <div type="textpart" subtype="explicit"><ab>
+                <l n="66">፷፮፡ ክሕደተ፡ ጴጥሮስ።</l>
+                <l n="67">፷፯፡ ነስሖተ፡ ይሁዳ።</l>
+                <l n="68">፷፰፡ ዘከመ፡ ነሥአ፡ ዮሴፍ፡ራማዊ፡ ሥጋሁ፡ ለእግዚእ።</l></ab></div>
+              </div>
+              <div type="textpart" subtype="chapter" xml:lang="gez" corresp="LIT2709Matthew">
+                <label>Bǝsrāta Mātewos</label>
+               <div type="textpart" subtype="subtextunit" xml:id="MatthewGospelHeadline">
+                 <label>Headline of the Gospel of Matthew</label>
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። <seg type="title inscriptio expanded">ብስራተ፡ ወንጌል፡
+                   ማቴዎስ፡ ሐዋርያ። አሓዱ፡ እምዓሠርቱ፡ ወክልኤቱ።</seg> ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ በእደ፡ ማርያም፡ ዘመለኮት።
+                    ለዓለመ፡ ዓለም፡ አሜን።</ab>
+                </div>
+                <div type="textpart" subtype="coretext" xml:id="MatthewGospel">
+                  <label>Gospel of Matthew</label>
+                  <div type="textpart" subtype="incipit"><ab>መጽሐፈ፡ ልደቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ።
+                  ወልደ፡ ዳዊት፡ ወልደ፡ አብርሃም። አብርሃምኒ፡ ወለዶ፡ ለይስሓቅ። ወይስሓቅኒ፡ ወለዶ፡ ለያዕቆብ።</ab></div>
+                  <div type="textpart" subtype="explicit"><ab>ወናሁ፡ አነ፡ ሀሎኩ፡ ምስሌክሙ፡ በኵሉ፡ መዋዕል፡ እስከ፡
+                  ኅልቀተ፡ ዓለም፡ አሜን።</ab></div>
+                </div>
+                <div type="textpart" subtype="colophon" xml:lang="gez" xml:id="PostscriptMatthew">
+                  <label>Colophon of the Gospel of Matthew</label>
+                  <div type="textpart" subtype="incipit"><ab>መልአ፡ ጽሕፈተ፡ <seg type="title subscriptio">ብስራቱ፡
+                  ለማቴዎስ፡ ሐዋርያ።</seg>ወኮነ፡ ጸሓፎ፡ በምድረ፡ ፍልስጥዔም፡</ab></div>
+                  <div type="textpart" subtype="explicit"><ab>በቀዳሚሁ፡
+                  ዓ<sic>ም</sic><corr resp="ABr">መ</corr><supplied reason="omitted" resp="ABr">ተ</supplied>፡ እምቀላውዴዎስ፡ ቄሳር፡ ንጉሠ፡
+                  ሮሜ። ጸሎቱ፡ ወበረከቱ፡ ለቅዱስ፡ ማቴዎስ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ዘመሎኮት፡ ለዓለመ፡ ዓለም፡ አሜን።</ab></div>
```

Comment:
ዘመለኮት፡
Anyway, is Zamalakot a proper name? If so, I think we can omit this information in the work record (it only concerns one specific manuscript) and replace that name with `<gap reason="ellipsis" resp="Abr"/>`